subscription_oca/views/sale_subscription_views.xml

```diff
@@ -370,12 +370,66 @@
         <field name="model">sale.subscription</field>
         <field name="arch" type="xml">
             <search>
+                <field
+                    name="name"
+                    filter_domain="['|', ('name','ilike',self), ('code','ilike',self)]"
+                />
                 <field name="to_renew" />
```

Comment:
**todo**: This one filter doesn't seem to show up, presumably because you can't use text search on a boolean field.

![image](https://github.com/OCA/contract/assets/78726989/556b07e2-6dd4-4665-89ef-5a106179843e)


I'd recommend removing it.